docs/api.md

```diff
@@ -47,6 +47,14 @@ starts with two extra elements:`process.execPath` and the path to the JavaScript
 file being executed. So if you’re getting your arguments from `process.argv` in
 Node, pass `process.argv.slice(2)` to Yargs.
 
+_Note: yargs exposes the helper `hideBin`, handles the `process.argv.slice
+logic for you._
```

Comment:
There are a few formatting and grammar problems in this paragraph. Also, I personally think the text looks more readable if only the “Note” part of the paragraph is italic (as the majority of notes in `api.md` are formatted). Here is my suggested replacement paragraph that makes all those changes:

```markdown
_**Note:**_ Yargs exposes the helper `hideBin`, which handles the `process.argv.slice`
logic for you.
```

> _**Note:**_ Yargs exposes the helper `hideBin`, which handles the `process.argv.slice` logic for you.

For comparison, here’s how it looks with all of my changes except making the paragraph non-italic:

> _Note: Yargs exposes the helper `hideBin`, which handles the `process.argv.slice` logic for you._